.ci/scripts/functional-test.sh

```diff
@@ -23,6 +23,9 @@ METRICBEAT_VERSION=${4:-'8.0.0-SNAPSHOT'}
 TARGET_OS=${GOOS:-linux}
 TARGET_ARCH=${GOARCH:-amd64}
 
+## Install the required dependencies for the given SUITE
+.ci/scripts/install-test-dependencies.sh "${SUITE}"
+
```

Comment:
It calls the script though the one in charge to retry is the `install-test-dependencies.sh` itself